models/select-all-comments-for-article.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../db/connection');
+
+exports.selectAllCommentsForArticle = async (article_id) => {
+    const comments = await db.query(
+        `SELECT c.comment_id, c.body, c.article_id, c.author, c.votes, c.created_at
+        FROM comments c
+        JOIN articles a
+        ON a.article_id = c.article_id
+        WHERE c.article_id = $1
+        ORDER BY c.created_at DESC`, [article_id]
+    );
+    return comments.rows;
```

Comment:
Don't think you'll need a join here as we aren't asking for any article columns.